node_modules/credscan-pkg/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "credScanEnable": false
```

Comment:
why it is false?

Reply:
why is it needed here? You have already added in cli action, I don't think it is required,

Reply:
I forgot to change that. I'll change it

Reply:
Removed the file